src/components/charts/util/ChartSelect.tsx

```diff
@@ -73,7 +74,13 @@ const ChartSelect: React.FC<ChartSelectProps> = ({ chart, selectChart, addChart,
                 <span className="chart-grid-item__title-repo">{chart.chart_name}</span>
                 <span>/{chart.name}</span>
             </div>
-            <div className="chart-grid-item__chart-version">{chart.version}</div>
+            <div className="flexbox" style={{alignItems: 'center'}}>
+                <div className="chart-grid-item__chart-version" style={{margin: '0 12px 0 0'}}>{chart.version}</div>
```

Comment:
use mr-12 from formulae.scss and similarly at other places

Reply:
Changed